### PR TITLE
0.6.16

### PR DESCRIPTION
**Pull request by mbostock**
https://github.com/observablehq/plot/blob/0.6.16/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,113 @@
...
+      strokeWidth: (i) => i ** 1.1 / 100,
+      markerEnd: "arrow"
+    })
+  ]
+})
+```
+
+This release includes a few more new features, bug fixes, and improvements:
+
+The new **className** [mark option](https://observablehq.com/plot/features/marks#mark-options) specifies an optional `class` attribute for rendered marks, allowing styling of marks via external stylesheets or easier selection via JavaScript; thanks, @RLesser! Plot now reuses `clipPath` elements, when possible, when then **clip** mark option is set to *frame* or *projection*.
```
Comment (Fil) [inline CHANGELOG.md:108]:
```suggestion
The new **className** [mark option](https://observablehq.com/plot/features/marks#mark-options) specifies an optional `class` attribute for rendered marks, allowing styling of marks via external stylesheets or easier selection via JavaScript; thanks, @RLesser! Plot now reuses `clipPath` elements, when possible, when the **clip** mark option is set to *frame* or *projection*.
```

**Review by Fil — COMMENTED** [review]
a typo

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,113 @@
...
+```js
+Plot.plot({
+  fx: {interval: 10},
+  color: {legend: true},
+  marks: [Plot.waffleY(olympians, Plot.groupZ({y: "count"}, {fill: "sex", sort: "sex", fx: "weight", unit: 10}))]
+})
+```
+
+
+All marks now support GeoJSON data and GeoJSON property shorthand, making it easier to work with GeoJSON. For example, below the data `counties` is a GeoJSON FeatureCollection, and `unemployment` refers to a property on each feature; the **fill** option is thus shorthand for `(d) => d.properties.unemployment`. The [geo mark](https://observablehq.com/plot/marks/geo) now also supports the **tip** option (via an implicit [centroid transform](https://observablehq.com/plot/transforms/centroid)), making it easier to use Plot’s [interactive tooltips](https://observablehq.com/plot/interactions/pointer).
```
Comment (Fil) [inline CHANGELOG.md:22]:
```suggestion
All marks now support GeoJSON property shorthand, making it easier to work with GeoJSON. For example, below the data `counties` is a GeoJSON FeatureCollection, and `unemployment` refers to a property on each feature; the **fill** option is thus shorthand for `(d) => d.properties.unemployment`. All marks now support GeoJSON data, so you can replace Plot.geo below with Plot.dot to make a cartogram (but you’ll need to specify a transform such as Plot.centroid to compute the x and y channels). The [geo mark](https://observablehq.com/plot/marks/geo) now also supports the **tip** option (via an implicit [centroid transform](https://observablehq.com/plot/transforms/centroid)), making it easier to use Plot’s [interactive tooltips](https://observablehq.com/plot/interactions/pointer).
```

Not sure if it's much better with this suggestion?

**Review by Fil — APPROVED** [review]
(My comment above is not blocking.)

File: CHANGELOG.md (hunk 2 above)
Comment (mbostock) [reply CHANGELOG.md:22]: I had a comment to that effect previously but I removed it because it didn’t seem worth the trouble.

**Pull request merged**